### ISSUE TITLE
diff file include the \input or \include even for commented lines

### ISSUE THREAD

**Issue by rototrolxtrol** [driquet/rcs-latexdiff#11]
Commented lines  that contain an  \input or \include command are included in the diff.pdf file.

I think that it's due to the code that recognize this commands don't check if it belongs to a commented line, and decide if it should be included or not. 

Again thanks in advance

**Comment by ihincks** [comment]
Yes, you are right, commented includes do not work. I also found two related bugs: (1) if two inputs are on the same line it is too greedy with the filename, and (2) the included file must have a .tex extension.

I have submitted a PR which fixes all three bugs.

**Comment by rototrolxtrol** [comment]
Hi, thanks for the fast response.

I changed three files  (rcs_lateddiff.py, rcs.py,utils.py) but the 'diff.tex' file cannot be compiled. Then I checked the .new and .old files which showed this,

_"...	% Input intro

% End of Input intro

		% Input related_work

% End of Input related_work ...."_

None of the files was included (no pdf file was compiled). I don't know if maybe I'm pull the wrong path in repository. Because I installed via _git clone https://github.com/driquet/rcs-latexdiff.git_  (but nothing happend) and then update the above python files.

Thanks,
